src/Interface/hiopInterface.hpp

```diff
@@ -59,6 +59,10 @@
 #define MPI_Comm int
 #endif
 
+#ifndef MPI_COMM_SELF
+#define MPI_COMM_SELF 0
+#endif
+
 #ifndef MPI_COMM_WORLD
 #define MPI_COMM_WORLD 0
 #endif 
```

Comment:
I believe this is implemented in `hiopMPI.hpp` header file. I suggest including it here rather than replicating similar logic.

Reply:
Cool. Didn't know that. But it seems like MPI_COMM_WORLD is missing in the hiopMPI.hpp. Shall we add it to that file? Thanks.

Reply:
@cnpetra suggested using `MPI_COMM_SELF` instead. That is why we don't have definition of `MPI_COMM_WORLD` in the `hiopMPI.hpp`.